parsing/g_constr.mlg

```
@@ -135,6 +135,9 @@ GRAMMAR EXTEND Gram
       | "SProp" -> { None, UNamed [CSProp, 0] }
       | "Type" -> { None, UAnonymous {rigid=UnivRigid} }
       | "Type"; "@{"; "_"; "}" -> { None, UAnonymous {rigid=UnivFlexible false} }
+      | "Type"; "@{"; "Set" ; "|"; u = universe; "}" -> { None, u }
+      | "Type"; "@{"; "Prop" ; "|"; u = universe; "}" -> { None, UNamed [CProp, 0] }
+      | "Type"; "@{"; "SProp" ; "|"; u = universe; "}" -> { None, UNamed [CSProp, 0] }
```

Set isn't even a quality, and the others silently drop the universe

---

We have a better proposal now, closing this